From 4fa12df4ef3b966a09e36932e76652c6ff0fa7f5 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Thu, 5 Sep 2019 09:08:48 -0700
Subject: [PATCH] dgit-maint-{merge,debrebase}(7): Use git fetch --all --tags

`git remote update` doesn't fetch tags unless remote branches include
the commits at which those tags point.  Thus, if upstream pushes their
release tag but fails to push their master branch, `git remote update`
will not fetch the release tag.

I've been in this situation more than once when following the
workflows detailed in these manpages, so let's just recommend a
command which will definitely try to fetch the latest release tag.

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 2 +-
 dgit-maint-merge.7.pod     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index 9c9598bb..27c97aa3 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -353,7 +353,7 @@ release, and importing that release using git-debrebase(1).
 
 =over 4
 
-    % git remote update
+    % git fetch --all --tags
 
 =back
 
diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index 0ccd8c7e..37a02613 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -362,7 +362,7 @@ to git), you can just run dpkg-buildpackage(1) or debuild(1) instead.
 
 =over 4
 
-    % git remote update
+    % git fetch --all --tags
 
 =back
 
-- 
2.20.1

